astropy/table/table.py

```diff
@@ -2132,6 +2132,43 @@ def remove_rows(self, row_specifier):
         if hasattr(self, '_groups'):
             del self._groups
 
+    def iterrows(self, col_names=None):
+        """
+        Return zipped columns for row-wise iteration
+
+        Parameters
+        ----------
+        col_names : list
+        List of column names
+
+         Examples
+        --------
+        Create a table with three columns 'a', 'b' and 'c'::
+
+            >>> t = Table([[1, 2, 3], [0.1, 0.2, 0.3], ['x', 'y', 'z']],
```

Comment:
Use float values like 1.0, 2.5, 5.0 that have exact float representation.  That prevents any possibility of docstring fails on different architectures.